docs/getting-started/pip_installing.md

```diff
@@ -9,10 +9,10 @@ Depending on your OS, Concrete ML may be installed with Docker or with pip:
 |                 OS / HW                 | Available on Docker | Available on pip |
 | :-------------------------------------: | :-----------------: | :--------------: |
 |                  Linux                  |         Yes         |       Yes        |
-|                 Windows                 |         Yes         |  Not currently   |
+|                 Windows                 |         Yes         |   Coming soon    |
```

Comment:
I don't think we'll ever support native windows (the compiler won't for some time). So maybe just put "No"